src/gitops/components/rollout/revisions/Revisions.scss

```
@@ -0,0 +1,6 @@
+.gitops-revisions-table {
+  tbody > tr:first-child {
+    --pf-v6-c-table__tr--BorderBlockEndWidth: 1px;
+    border-top: 1px solid;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)